styles/mixins/triangle.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+module.exports = function( config ) {
```

Comment:
Or just use a right facing triangle and hide it behind the tab.  That would work.